### PR TITLE
[incubator/elasticsearch] added plugin install option

### PR DESCRIPTION
**Pull request by monotek**
**What this PR does / why we need it**: 
* adds option to install elasticsearch plugins like ingest-attachment, mapper-size and so on...

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by davidkarlsen** [comment]
Isn't it better to have the image pre-built with plugins already present?

**Comment by monotek** [comment]
Kind of. But you would have a lot of possible plugin combinations and therefore would need a lot of different images.

**Comment by mattfarina** [comment]
@simonswine @icereval @rendhalver Can you please review this since it adds a feature around plugins.

File: incubator/elasticsearch/README.md [hunk 1]
```diff
@@ -113,6 +113,9 @@ The following table lists the configurable parameters of the elasticsearch chart
 | `data.tolerations`                   | Data tolerations                                                    | `{}`                                 |
 | `data.terminationGracePeriodSeconds` | Data termination grace period (seconds)                             | `3600`                               |
 | `data.antiAffinity`                  | Data anti-affinity policy                                           | `soft`                               |
+| `plugins.command`                    | Path to Elasticsearchs plugin command                               | `"/usr/share/elasticsearch/bin/elasticsearch-plugin"` |
```
Comment (rendhalver) [inline incubator/elasticsearch/README.md]: I am still not sure about doing this.
This goes against the immutability of containers.

**Comment by rendhalver** [comment]
I think installing plugins into elasticsearch goes against the immutability of containers.
If people need plugins it's really easy to build a custom container and use that.
It's quicker to deploy that was as well.

**Comment by rendhalver** [comment]
Can we get elasticseach promoted to stable before we do this?
I have another PR to add myself to the OWNERS file that would be good to merge as well so we can speed up the approval process.

**Comment by monotek** [comment]
@rendhalver 
Are you asking for the move to stable inside this pr?

**Comment by rendhalver** [comment]
@monotek I have a PR to promote elasticsearch to stable.
I don't think we should be doing plugin installs for elasticsearch.
It goes against the immutability of containers.
See the discussion in #5651

**Comment by monotek** [comment]
ok, i'll close it...

**Pull request closed without merging**